src/content/brand.md

```diff
@@ -16,8 +16,10 @@ material, as determined by Google.
 Use of the Dart trademarks that is not expressly permitted by these
 guidelines is prohibited absent written permission from Google.
 
-The official Dart logos can be found in the [Dart and Flutter logo
-assets](https://drive.google.com/corp/drive/folders/1KXNtO9My2AMpDOF9A9Y_4aj4_BcgmDDT) Drive folder.
+The official Dart logos can be found in the
+[Dart and Flutter logo assets]
+(https://services.google.com/fh/files/misc/dart_brand_guidelines_assets.zip)
```

Comment:
Renders like:
<img width="720" alt="image" src="https://github.com/dart-lang/site-www/assets/111139605/2bfe0924-139c-44ac-8c49-16aa4026aa62">

Staging here: https://dart-dev--pr5906-fix-5904-6vt1xvms.web.app/brand

I think they need to be on the same line:
```suggestion
[Dart and Flutter logo assets](https://services.google.com/fh/files/misc/dart_brand_guidelines_assets.zip)
```